examples/03_connectivity/plot_simulated_connectome.py

```diff
@@ -7,16 +7,15 @@
 
 """
 
-import matplotlib.pyplot as plt
-
 
 def plot_matrix(m, ylabel=""):
     abs_max = abs(m).max()
     plt.imshow(m, cmap=plt.cm.RdBu_r, interpolation="nearest",
                vmin=-abs_max, vmax=abs_max)
 
-
+###########################################################################
```

Comment:
Actually, this example needs to run in a single cell, as it displays all the information on the same figure (see the rendering). For this specific example, we need to revert.